### ISSUE TITLE
Interacting with blocks does not utilise /ctb

### ISSUE THREAD

**Issue by xFier** [CivMC/Civ#542]
Interacting with reinforced dirt, grass, or logs to make dirt paths/stripped logs does not utilise /ctb, meaning whoever is breaking the block has to break it one at a time to change the block into a dirt path/stripped log.

_This might also affect tilling for farms._

Steps to reproduce:
1. Place reinforced dirt, grass or log
2. Wait until it's matured
3. Right click it with a shovel/axe to make a dirt path or stripped log

Basically, you should be able to make a dirt path/stripped log without breaking the entire reinforcement.
